src/main/java/com/unityTest/testrunner/service/CodeService.java

```diff
@@ -183,4 +199,26 @@ private String buildPythonTestFunction(String funcName, String body) {
 		// Create code block by indenting the body by one tab
 		return def.concat(Utils.indent(body, 1).concat("\n"));
 	}
+
+	private String buildHUnitTestFunction(String funcName, String body) {
+		StringBuilder resultBuilder = new StringBuilder();
+
+		// Importing Hunit
+		resultBuilder.append("import HUnit");

```

Comment:
Also you can leave out the HUnit import, they can provide that in the suite file.